pvm/ts/src/engine/texasHoldem2.test.ts

```
@@ -405,15 +405,14 @@ describe.only("Texas Holdem Game", () => {
             expect(game.getPlayerCount()).toEqual(2);
         });
 
-        it("should handle betting actions", () => {
-            const player = game.getPlayer("0x1fa53E96ad33C6Eaeebff8D1d83c95Fcd7ba9dac");
-
-            // // Test different actions
-            // game.performAction("0x1fa53E96ad33C6Eaeebff8D1d83c95Fcd7ba9dac", PlayerActionType.CHECK);
-            // expect(game.getPlayersLastAction("0x1fa53E96ad33C6Eaeebff8D1d83c95Fcd7ba9dac")?.action).toEqual(PlayerActionType.CHECK);
+        it.only("should validate legal actions after blinds", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)